gateway-messages/tests/versioning/v6.rs

```diff
@@ -0,0 +1,99 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! The tests in this module check that the serialized form of messages from MGS
+//! protocol version 5 have not changed.
```

Comment:
```suggestion
//! protocol version 6 have not changed.
```

Reply:
Fixed in https://github.com/oxidecomputer/management-gateway-service/pull/92/commits/1879755cbe398073368338d11717a8ac8dc7fd93